packages/expo-cli/src/commands/eas-build/build/utils/git.ts

```diff
@@ -1,12 +1,51 @@
 import spawnAsync from '@expo/spawn-async';
 import fs from 'fs-extra';
 import ora from 'ora';
+import which from 'which';
```

Comment:
there is already `command-exists` package added to expo-cli deps that could be used here

https://github.com/expo/expo-cli/blob/master/packages/expo-cli/src/credentials/views/SetupAndroidKeystore.ts